examples/sampling-slice-adaptive-hyperrectangles-mcmc.ipynb

```diff
@@ -0,0 +1,948 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x10d575c88>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import os\n",
```

Comment:
Please install PINTS, using e.g. `pip install -e .[dev,test]` (see the README and CONTRIBUTING).
After that these lines `import os` and `os.chdir` won't be necessary anymore and can be removed